experimental/nodejs-functions/stack.yaml

```diff
@@ -1,7 +1,9 @@
-id: nodejs-functions
-name: Node.js Functions application
+name: Node.js Functions
 version: 0.1.0
-description: Simple Node.js Functions application with health checks and application metrics for Prometheus.
-maintainer: Chris Bailey <cnbailey@gmail.com>
-default-template: simple
+description: Serverless Node.js functions
```

Comment:
can we add Express here?

> Node.js Express Functions

Reply:
or something that implies that this stack in inherited from the Node.js Express stack..

Reply:
I'm thinking this is what the user sees on the website tile for the stack. I don't think it matters to the user that it is based on express.. am I missing something?